### PR TITLE
LPS-167759 Resources Folder in export/import of fragments

### PR DESCRIPTION
**Pull request by jkappler**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:sf

**Comment by ealonso** [comment]
Just started reviewing :)

File: modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/FragmentCollectionResourcesDisplayContext.java [hunk 1]
```diff
@@ -67,6 +70,34 @@ public long getFolderId() throws PortalException {
...
 
+	public long getRepositoryId() throws PortalException {
+		if (_repositoryId != null) {
+			return _repositoryId;
+		}
+
+		FragmentCollection fragmentCollection =
+			_fragmentDisplayContext.getFragmentCollection();
+
+		Repository repository =
```
[the remark this answers is not in the document]
Comment (jkappler) [reply modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/FragmentCollectionResourcesDisplayContext.java:81]: @ealonso, it calls the other method

@Override
	public long getResourcesFolderId() throws PortalException {
		return getResourcesFolderId(true);
	}

[the remark this answers is not in the document]
Comment (jkappler) [reply modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/FragmentCollectionResourcesDisplayContext.java:81]: @ealonso, @lfbesada, but we need to retrieve the folder again, so my idea would be to create the repository and folder when creating the collection

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**